.github/workflows/ci-pkg-install.yml

```diff
@@ -17,141 +17,40 @@ defaults:
 
 jobs:
 
-  init-temp:
-    runs-on: ubuntu-20.04
-    steps:
-    - run: mkdir pypi && touch pypi/.placeholder
-    - uses: actions/upload-artifact@v3
-      with:
-        name: ci-packages-${{ github.sha }}
-        path: pypi
-
-  install-standalone:
-    needs: init-temp
+  install-pkg:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
-      max-parallel: 1
+      fail-fast: false
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04, macOS-11, macOS-12, windows-2022]
-        pkg: ["app", "lite", "pytorch"]
-        python-version: [3.8]  # , 3.9
+        pkg: ["app", "lite", "pytorch", ""]
+        python-version: ["3.7" , "3.10"]
 
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install pytest extensions
+        # TODO: for mono-package create & load requirements/test.txt
+        if: ${{ matrix.pkg != '' }}
+        run: grep 'pytest>' requirements/${{ matrix.pkg }}/test.txt | xargs -0 pip install
+
```

Comment:
Why do we need pytest extensions? This job only tests setup-tools and checks the lightning version

Reply:
```suggestion
```
good question, but it was there already, not addition of this PR